mava/configs/system/q_learning/rec_iql.yaml

```
@@ -11,13 +11,13 @@ add_agent_id: True
 min_buffer_size: 32
 update_batch_size: 1 # Number of vectorised gradient updates per device.
 
-rollout_length: 2 # Number of environment steps per vectorised environment.
+rollout_length:  2 # Number of environment steps per vectorised enviro²nment.
```

Not sure how the ^2 got in there :joy: 
```suggestion
rollout_length: 2 # Number of environment steps per vectorised environment.
```